packages/flutter_test/lib/src/binding.dart

```diff
@@ -582,6 +597,42 @@ class AutomatedTestWidgetsFlutterBinding extends TestWidgetsFlutterBinding {
     });
   }
 
+  @override
+  Future<T> runAsync<T>(Future<T> callback()) {
+    assert(() {
+      if (_pendingAsyncTasks == null)
+        return true;
+      throw new test_package.TestFailure(
+          'runAsync() was called, then before its future completed, it\n'
+          'was called again. You must wait for the first returned future\n'
+          'to complete before calling runAsync() again.'
```

Comment:
We don't normally break lines for `FlutterError`s (see https://master-docs-flutter-io.firebaseapp.com/flutter/foundation/FlutterError/message.html for the conventions), but I don't know what the conventions are for `TestFailure`.

Reply:
Done